src/server/scripts/Events/brewfest.cpp

```diff
@@ -2038,7 +2038,7 @@ class spell_direbrew_disarm : public AuraScript
         if (Aura* aura = GetTarget()->GetAura(SPELL_DIREBREW_DISARM_GROW))
         {
             aura->SetStackAmount(aura->GetStackAmount() + 1);
-            aura->SetDuration(aura->GetDuration() - 1500);
+            aura->SetDuration(Milliseconds(aura->GetDuration()) - 15s);
```

Comment:
@Maelthyrr this doesnt seem right - could case a crash if negative

Reply:
You're right. But does some aura durations can be negative ?

Reply:
from an ingame perspective: no
code-wise: doubt.

maybe, it makes sense in the broader scope, to add some logic checks to functions (i think we have had crash issues with a random scheduler that had to select between a lower value and an higher value, but accidentally, the higher value was smaller than the lower value) and overloaded functions that prevent negative duration